core/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -112,30 +116,40 @@ public boolean allowAliasesToMultipleIndices() {
     }
 
     public void writeIndicesOptions(StreamOutput out) throws IOException {
-        out.write(id);
+        out.writeInt(id);
     }
 
     public static IndicesOptions readIndicesOptions(StreamInput in) throws IOException {
         //if we read from a node that doesn't support the newly added flag (allowAliasesToMultipleIndices)
         //we just receive the old corresponding value with the new flag set to true (default)
```

Comment:
may I ask you to remove this comment too? it's about bw comp which we have dropped a while ago